### DISCUSSION TITLE
A repository of content and templates (posters, banners, stickers, etc.) for protests worldwide

### DISCUSSION THREAD

**Discussion by noidsirius** [WeAreMahsaAmini/Core#84] [Ideas]
- The content can be searchable (probably by hashtags)
- (Preferably) Different formats should be provided for convenient reuse in different occasions

**Comment by karimifar** [comment]
I started collecting posters/signs/artworks on instagram in a google drive folder. created an email so people can send their designs. It's pretty helpful for those outside iran going to protests and needing to print something or just wanting to post a social media post...

**Reply by sammigh** [reply]
could u please send the link to me? need to print some for Saturday

**Reply by karimifar** [reply]
https://drive.google.com/drive/u/2/folders/1DgRBz8EFMHHuo4SXqeZaeOsgCKTvewIo

**Reply by sammigh** [reply]
Thanks, may I share it with my freinds?

**Reply by karimifar** [reply]
It is meant to be shared! For the artworks folder, please credit the artist. Their instagram handle is in the name of the file. 
We are planning to add a folder for photos. Stay tuned!

**Reply by sammigh** [reply]
Sure, Thanks :)

**Reply by anonymooshia** [reply]
This is wonderful. What kind of photos are you looking to publish? Perhaps I can help. Personally I'd love to have the defining photos of the protests so far. Could be great for creating video content with various music/video to get stuff going viral.

**Reply by karimifar** [reply]
We've added an announcement in the folder. That's a great idea and we can create a subfolder for defining moments. We could use your help. How can I reach out to you?

**Reply by sammigh** [reply]
@karimifar There are amazing artworks on below Telegram page (we are printing them on T-shirts for tomorrow rally ) Might be good to add them to your archive 
https://t.me/NaghmehJahTshirts

**Comment by ramyareye** [comment]
we can get ideas from 
https://supportukrainenow.org/ 
https://helpukraine.center/ 
https://war.ukraine.ua/support-ukraine/
